Bundle/UKCore-Bundle-MichaelChalmersRequest-Transaction-Example.json

```diff
@@ -0,0 +1,764 @@
+            "url": "urn:uuid:8d490f90-e24f-428d-8d9d-aecd9e699087"
+         }
+      },
+      {
+         "fullUrl": "urn:uuid:d2a0760e-6e34-45b6-8cd0-5416a4bfa630",
+         "resource": {
+            "resourceType": "Specimen",
+            "id": "d2a0760e-6e34-45b6-8cd0-5416a4bfa630",
+            "accessionIdentifier": {
+               "system": "https://www.ouh.nhs.uk//path",
+               "value": "Spec123456"
+            },
+            "status": "available",
+            "type": {
+               "coding": [
+                  {
+                     "system": "http://snomed.info/sct",
+                     "code": "87612001",
+                     "display": "Blood (substance)"
+                  }
+               ]
+            },
+            "subject": {
+               "reference": "urn:uuid:9fd5cd87-3668-4946-a109-6e54d0a9df3b"
+            },
+            "request": [
+               {
+                  "reference": "urn:uuid:490519b8-3724-4fe7-8367-9f859e7786b4"
+               }
+            ],
+            "collection": {
+               "collectedDateTime": "2022-07-11T09:00:00Z"
+            }
+         },
+         "request": {
+            "method": "POST",
+            "url": "urn:uuid:d2a0760e-6e34-45b6-8cd0-5416a4bfa630"
+         }
+      },
+      {
+         "fullUrl": "urn:uuid:dcc6e70d-e97a-4078-a696-c522a30cc552",
+         "resource": {
+            "resourceType": "Condition",
+            "id": "dcc6e70d-e97a-4078-a696-c522a30cc552",
+            "clinicalStatus": {
+               "coding": [
+                  {
+                     "system": "http://terminology.hl7.org/CodeSystem/condition-clinical",
+                     "code": "active",
+                     "display": "Active"
+                  }
+               ]
+            },
+            "verificationStatus": {
+               "coding": [
+                  {
+                     "system": "http://terminology.hl7.org/CodeSystem/condition-ver-status",
+                     "code": "provisional",
+                     "display": "Provisional"
+                  }
+               ]
+            },
+            "code": {
+               "coding": [
+                  {
+                     "system": "http://snomed.info/sct",
+                     "code": "39925003",
+                     "display": "Juvenile myopathy, encephalopathy, lactic acidosis AND stroke"
+                  }
+               ]
+            },
+            "subject": {
+               "reference": "urn:uuid:9fd5cd87-3668-4946-a109-6e54d0a9df3b"
+            },
+            "onsetDateTime": "2022-06-13T09:00:00Z",
+            "recordedDate": "2022-11-13T09:00:00Z"
+         },
+         "request": {
+            "method": "POST",
+            "url": "urn:uuid:dcc6e70d-e97a-4078-a696-c522a30cc552"
+         }
+      },
+      {
+         "fullUrl": "urn:uuid:585bd4e4-8e50-4adf-a333-d1ea9f594fe9",
+         "resource": {
+            "resourceType": "Condition",
```

Comment:
Add Condition.note covering the note in the text box: "Spasticity and mobility problems since 2018"